core/primitives/src/views.rs

```diff
@@ -215,10 +215,8 @@ impl From<AccessKeyView> for AccessKey {
 /// Item of the state, key and value are serialized in base64 and proof for inclusion of given state item.
 #[derive(serde::Serialize, serde::Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct StateItem {
-    #[serde(with = "base64_format")]
-    pub key: Vec<u8>,
-    #[serde(with = "base64_format")]
-    pub value: Vec<u8>,
+    pub key: StoreKey,
+    pub value: StoreValue,
```

Comment:
Note: This looks like a breaking change for the published primitives crate

Reply:
@khorolets would this be a problem? Serialized, the data should look the same, it's only a breaking change on the code level, as constructors needs to be updated.

Reply:
Doesn't seem like a problem for us. What nearcore version are these changes expected to land at? I want us to check the underlying stuff and ensure we do not lose data or prevent services from being stuck 😬

Reply:
@khorolets it will land in neard binary version 1.35 if we merge it within the next couple of weeks. For the crates version, I guess we will have to bump one from the current version and would end up with 0.17.

Reply:
Sounds good to me